basics/keeping-up-to-date/upgrade.md

```diff
@@ -49,7 +49,7 @@ zip file itself, which must be extracted as well.
 On a Linux terminal, you can use the command \`unzip\`:
 
 ```bash
-unzip prestashop-upgrade.zip && unzip prestashop.zip
+unzip prestashop_8.0.1.zip && unzip prestashop_8.0.1.zip
```

Comment:
Why unzip twice the same file ? Before it was about unzipping two different files (one being the one downloaded in L39)